image-set-polyfill.js

```diff
@@ -106,7 +106,7 @@
      * @return {String} new styles
      */
     function getPolyfilledStyles(styles) {
-        var findImageSetRE = /(?:\-(?:webkit|moz)\-)?image\-set\(((?:a|[^a])*)\)/mg,
+        var findImageSetRE = /(?:\-(?:webkit|moz)\-)?image\-set\(((?:a|[^a;}])*)\)/mg,
```

Comment:
`(?:a|[^a])` means `everything` but if we are going to exclude the `;` we can get rid of this stange constructing a replace it with
`(?:a|[^a;])` -> `[^;]`